projects/cognito-service/src/lib/cognito.service.ts

```diff
@@ -676,6 +676,11 @@ export class CognitoService
   // NOTE: Authentication ----------------------------------------------------------------------
   // -------------------------------------------------------------------------------------------
 
+  public login(provider : string, username ?: string, password ?: string) : Observable<any>
```

Comment:
Ok for the naming convention but we'll use this one instead : `signIn`, `signOut` & `signUp`.
So remove this function and rename `authenticateUser` to `signIn` (bellow and in documentation)

Reply:
Also in test project 😉